packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js

```diff
@@ -75,126 +92,128 @@ const ApplicationInfosPage = () => {
+                            defaultMessage: 'Upgrade your admin panel',
+                          })}
+                        </Link>
+                      )}
+                    </GridItem>
+                    <GridItem col={6} s={12}>
                       <Link
-                        href={`https://github.com/strapi/strapi/releases/tag/${latestStrapiReleaseTag}`}
+                        href="https://strapi.io/pricing-self-hosted"
                         isExternal
                         endIcon={<ExternalLink />}
                       >
                         {formatMessage({
-                          id: 'Settings.application.link-upgrade',
-                          defaultMessage: 'Upgrade your admin panel',
+                          id: 'Settings.application.link-pricing',
+                          defaultMessage: 'See all pricing plans',
                         })}
                       </Link>
-                    )}
-                  </GridItem>
-                  <GridItem col={6} s={12}>
-                    <Link
-                      href="https://strapi.io/pricing-self-hosted"
-                      isExternal
-                      endIcon={<ExternalLink />}
-                    >
+                    </GridItem>
+                  </Grid>
+
+                  <Box paddingTop={1}>
+                    <Typography variant="sigma" textColor="neutral600">
                       {formatMessage({
-                        id: 'Settings.application.link-pricing',
-                        defaultMessage: 'See all pricing plans',
+                        id: 'Settings.application.node-version',
+                        defaultMessage: 'node version',
                       })}
-                    </Link>
-                  </GridItem>
-                </Grid>
-
-                <Box paddingTop={1}>
-                  <Typography variant="sigma" textColor="neutral600">
-                    {formatMessage({
-                      id: 'Settings.application.node-version',
-                      defaultMessage: 'node version',
-                    })}
-                  </Typography>
-                  <Typography as="p">{appInfos.nodeVersion}</Typography>
-                </Box>
-              </Stack>
-            </Box>
-            {data && (
-              <CheckPermissions permissions={permissions}>
-                <Form ref={inputsRef} projectSettingsStored={data} />
-              </CheckPermissions>
-            )}
-          </Stack>
-        </ContentLayout>
+                    </Typography>
+                    <Typography as="p">{appInfos.nodeVersion}</Typography>
+                  </Box>
+                </Stack>
+              </Box>
+              {canRead && data && (
+                <Form canUpdate={canUpdate} ref={inputsRef} projectSettingsStored={data} />
+              )}
+            </Stack>
+          </ContentLayout>
+        </form>
```

Comment:
A form in a form? 🤔

Reply:
not a good naming indeed 🤔 wdyt of `Customization` component? or if you have another idea 😄